app.py

```diff
@@ -1,6 +1,4 @@
-# read .env files
-import dotenv, os
-dotenv.load_dotenv()
+import os

```

Comment:
The env vars will be configured for cloud run deployment, we don't use .env in cloud